manifests/defaults.pp

```diff
@@ -30,4 +30,10 @@
   else
   { $server_daemonize = true }
 
+  if defined('$is_pe') and str2bool($::is_pe) {
+    $ruby_interpreter = '/opt/puppet/bin/ruby'
+  } else {
+    $ruby_interpreter = '/usr/bin/env ruby'
+  }
+
```

Comment:
@buzzdeee Would it make sense to default $ruby_interpreter to something else when osfamily=OpenBSD ?

I don't know OpenBSD though, so maybe not.  There's a rubyversion fact available if that helps.